UserGuide.md

```diff
@@ -0,0 +1,15 @@
+Feature: Make a post anonymous
+How to use the feature:
+    Making a post anonymous:
+    - make a new post
+    - click the settings button to access the post tools
```

Comment:
might be more accurate to say the post drop down cause I think there's a separate settings button

Reply:
changed wording in newest commit